modules/storytelling/proto/story.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto2";
+
+package apollo.storytelling;
+
+import "modules/common/proto/header.proto";
+
+// TODO(xiaoxq): State triggering conditions.
+message ProceedWithCaution {
+}
+
+// TODO(xiaoxq): State triggering conditions.
+message CloseToJunction {
+  optional int32 distance = 1;
+}
+
+// Usage guide for action modules:
+// 1. Call `stories.has_XXX()` to check if a story you are interested is in
+//    charge.
+// 2. Access the story details if necessary, and take action accordingly.
+message Stories {
+  optional apollo.common.Header header = 1;
+
+  optional ProceedWithCaution proceed_with_caution = 2;
```

Comment:
Is it better to use Enum here instead? The scenarios/states should be exclusive and only one shall be active at one time instance.

Reply:
Per offline discussion, keep it as of now.